mainpage.hpp

```diff
@@ -0,0 +1,8 @@
+/*! \mainpage OmniPhotos Documentation
+ *
+ * \image html teaser.jpg
+ */
+
+/**
+ * @brief  Main application that displays OmniPhotos and handles user input.
+ */
```

Comment:
This probably belongs to the `ViewerGUI` class, no?

Reply:
How come? This is the code needed to customize Doxygens index page.

Reply:
My understanding is that the lines above these 3 are for customising the Doxygen main page. Could you please check they still work? https://omniphotos.readthedocs.io/en/latest/ is back to a blank page.

These three lines sound more like they are for a specific class (like ViewerApp or ViewerGUI), not the whole code base, which also includes the preprocessing.

Reply:
Aha, you are correct thanks! Will update this again. Must've messed up when copy-pasting.

Reply:
cecaa8a should fix the issue with the blank mainpage.

Reply:
Looks good now :+1: